app/models/address.rb

```diff
@@ -0,0 +1,16 @@
+class Address < ApplicationRecord
+  include ProviderObjectMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id
```

Comment:
Since an address is always expected to belong to something else, it shouldn't directly belong to an `ext_management_system`. The same way `Endpoint` doesn't define an `ext_management_system`.
Also when we are scanning them in inventory, we don't get them independantly but rather through their containers. So again, no need to directly belong to an EMS.